lib/ansible/plugins/inventory/gcp_compute.py

```diff
@@ -0,0 +1,298 @@
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: gcp_compute
+    plugin_type: inventory
+    short_description: Google Cloud Compute Engine inventory source
+    requirements:
+        - requests >= 2.18.4
+        - google-auth >= 1.3.0
+    extends_documentation_fragment:
+        - gcp
+        - constructed
+        - inventory_cache
+    description:
+        - Get inventory hosts from Google Cloud Platform GCE.
+        - Uses a <name>.gcp.yaml (or <name>.gcp.yml) YAML configuration file.
+    options:
+        zones:
+          description: A list of regions in which to describe GCE instances.
+        projects:
+          description: A list of projects in which to describe GCE instances.
+        filters:
+          description: A dictionary of filter value pairs. Available filters are listed here
```

Comment:
Please also add whether filters that are listed in separate list items are an `OR` or an `AND`

Reply:
Done.